src/layouts/applicationLayout.js

```diff
@@ -23,20 +37,23 @@ const applicationLayout = () => {
             </div>
         </div>
         <div className="wl_sp_lp_main_form_progress">
-            <button className="wl_sp_lp_main_form_btn_filled wl_sp_lp_main_form_btn">Application form</button>
+            <button className={tab===0?"wl_sp_lp_main_form_btn_filled":"wl_sp_lp_main_form_btn"} onClick={handleChange}>Application Form</button>
```

Comment:
Also try to find component for tabs as well in mui and if it doesn't exist (which is mp not the case) then you can install another ui library as well

Reply:
I'll try to replace it with stepper component of mui.